data/data-pipeline/data_pipeline/etl/score/etl_score_post.py

```diff
@@ -217,28 +219,8 @@ def _create_score_data(
             right_on=self.STATE_CODE_COLUMN,
             how="left",
         )
-
-        # check if there are census tracts without score
-        logger.info("Removing tract rows without score")
-
-        # merge census tracts with score
-        merged_df = national_tract_df.merge(
-            score_county_state_merged,
-            on=self.GEOID_TRACT_FIELD_NAME,
-            how="left",
-        )
-
-        # recast population to integer
-        score_county_state_merged["Total population"] = (
-            merged_df["Total population"].fillna(0).astype(int)
-        )
-
-        de_duplicated_df = merged_df.dropna(
-            subset=[DISADVANTAGED_COMMUNITIES_FIELD]
-        )
-
         # set the score to the new df
-        return de_duplicated_df
+        return score_county_state_merged
```

Comment:
we should confirm that there's no duplication, either here or on merge

Reply:
Great call, added.